ecllibrary/std/File.ecl

```diff
@@ -358,9 +358,10 @@ EXPORT RfsAction(varstring server, varstring query) :=
  * @param location      The IP address of the remote machine.
  * @param frompath      The path/name of the file to move.
  * @param topath        The path/name of the target file.
+ * @param planename     The name of the data plane containing the file.
  */
-EXPORT MoveExternalFile(varstring location, varstring frompath, varstring topath) :=
-    lib_fileservices.FileServices.MoveExternalFile(location, frompath, topath);
+EXPORT MoveExternalFile(varstring location, varstring frompath, varstring topath, varstring planename) :=
+    lib_fileservices.FileServices.MoveExternalFile(location, frompath, topath, planename);
```

Comment:
making it non-optional is going to break existing ECL.
When these (plane names) were introduced to the spray calls they were added as optional parameters for that reason.